tests/test_validation.py

```diff
@@ -665,6 +695,49 @@ def test_wrong_filetype(self):
             target_file, required_fields=['field_a'], excel_ok=False)
         self.assertTrue('could not be opened as a CSV' in error_msg)
 
+    def test_slow_to_open(self):
+        """Test timeout by mocking a CSV that is slow to open"""
+        from natcap.invest import validation
+
+        # make an actual file so that `check_file` will pass
+        path = os.path.join(self.workspace_dir, 'slow.csv')
+        with open(path, 'w') as file:
+            file.write('1,2,3')
+
+        # define a side effect for the mock that will sleep
+        # for longer than the allowed timeout
+        def delay(*args, **kwargs):
+            time.sleep(10)
+
```

Comment:
Same here ... between the `time.sleep` and the timeout we can set, would it be reasonable to turn down the expected runtime of these tests?  Just hoping to have our tests run as quickly as is reasonable, is all!